### DISCUSSION TITLE
Ajouter un gestionnaire à plusieurs cantines en même temps

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1162] [Idées]
Est-ce possible, depuis un compte admin, d'ajouter un utilisateur dans la liste des gestionnaires à plusieurs cantines en une seule fois?

Je m'explique : au lieu d'aller sur la page de chacune des cantines et de rajouter à chaque fois l'utilisateur B dans la liste des gestionnaires, j'irais sur la page du gestionnaire A qui a créé les cantines, je sélectionnerais toutes les cantines, je cliquerais sur un bouton du genre "Ajouter un gestionnaire", et je sélectionnerais l'utilisateur B.

Utilité : gain de temps lorsqu'un gestionnaire de cuisine centrale qui gère un nombre important de cantines satellites me demande d'ajouter un de ses collègues comme gestionnaire additionnel (même si évidemment il peut le faire lui-même pour chacune des cantines depuis son compte utilisateur).

**Comment by hfroot** [comment]
C'est pas possible pour l'instant. Pourquoi est-ce que le gestionnaire ne veut pas le faire lui-même ?

**Reply by ghost** [reply]
En fait, comme je sentais qu'ils étaient un peu réticents, c'était un argument de ma part : vous savez quoi, vous vous occupez de créer les diag et, de mon côté, je vous fais gagner du temps en rajoutant vous deux comme gestionnaires additionnels. Mais je pense que cette manip sera occasionnelle, voire rare.
